code/reports/rmdchunks/save-scores.Rmd

```diff
@@ -0,0 +1,7 @@
+<!--- Save evaluation scores as csv -->
+```{r, cache = TRUE, cache.extra = params$report_date}
```

Comment:
:red_circle: 

I don't think this should be cached(?)

Reply:
This was to prevent the `write_csv` call if there was no update.

Reply:
As it's not a computationally intensive, I'd rather do this step again even if it's not necessary (which shouldn't be happen in the normal use case). 

[As the developer of knitr keeps repeating](https://yihui.org/en/2018/06/cache-invalidation/), caching and caching invalidation are tricky and it's easy to get bitten with something that doesn't get updated when it should.

In this case for example, I **think** we don't invalidate the cache often enough because any change in `table` (e.g., if we changed the chunks above) will not trigger a re-run of this chunk. To get the desired behaviour, we would need to add another dependency to `table` in `cache.extra` or `autodep = TRUE` ([the default seems to be `FALSE`](https://yihui.org/knitr/options/#cache)).

Reply:
This makes sense - happy to remove.

Reply:
addressed in 9b293b6